express/app.js

```diff
@@ -25,6 +25,10 @@ module.exports = appInsights => express.Router()
     controllers.payhubController.postPaymentGroup(req, res, appInsights);
   })
 
+  .post('/payment-history/payment-groups/card-payments', (req, res) => {
```

Comment:
I think we are still using the old telephony endpoint [/payment-groups/{payment-group-reference}/card-payments]
Is this the same or a new one?